includes/rrdtool.inc.php

```diff
@@ -294,3 +298,17 @@ function rrdtool_escape($string, $maxlength=null){
     return $result.' ';
 
 }
+
+function rrdtool_tune($type, $filename, $max) {
+    if ($type === 'port') {
+        if ($max < 10000000) {
+            return false;
+        }
+        $max = $max / 8;
+        $fields = array(
+'INOCTETS','OUTOCTETS','INERRORS','OUTERRORS','INUCASTPKTS','OUTUCASTPKTS','INNUCASTPKTS','OUTNUCASTPKTS','INDISCARDS','OUTDISCARDS','INUNKNOWNPROTOS','INBROADCASTPKTS','OUTBROADCASTPKTS','INMULTICASTPKTS','OUTMULTICASTPKTS'
+);
+        $options = "--maximum " . implode(":$max --maximum ", $fields). ":$max";
+        rrdtool('tune', $filename, $options);
```

Comment:
Can you move this rrtool & $options statement outside of the `if ($type === 'port') {` block and add a `if (!empty($options)) {` block?
It will spare us redundant $options concat and rrdtool calls if we add more than just `port` :)